project.clj

```diff
@@ -7,12 +7,12 @@
 
   :dependencies []
 
-  :profiles {:dev {:source-paths ["dev"]
+  :profiles {:dev {:source-paths ["dev" "test/clj" "test/cljc"]
                    :dependencies [[org.clojure/clojure "1.9.0" :scope "provided"]
                                   ;; Add-ons:
                                   [org.clojure/core.async "0.4.474"]
                                   [manifold "0.1.8"]
-                                  [funcool/promesa "1.9.0"]
+                                  [funcool/promesa "1.10.0-SNAPSHOT"]
```

Comment:
Why an unreleased version?

Reply:
This is because promesa hasn't release a patch of the problem I described in one of the comments: https://github.com/funcool/promesa/issues/58